tests/dvslib/dvs_acl.py

```diff
@@ -87,7 +87,7 @@ def verify_acl_group_member(self, acl_group_ids, acl_table_id):
         member_groups = []
         for member in members:
             fvs = self.asic_db.wait_for_entry("ASIC_STATE:SAI_OBJECT_TYPE_ACL_TABLE_GROUP_MEMBER", member)
-            for k, v in fvs.items():
+            for k, v in list(fvs.items()):
```

Comment:
Do you know why this change was necessary? I thought `items()` was compatible with python3.

Reply:
In Python 2, the methods items(), keys() and values() used to "take a snapshot" of the dictionary contents and return it as a list. It meant that if the dictionary changed while you were iterating over the list, the contents in the list would not change.

In Python 3, these methods return a view object whose contents change dynamically as the dictionary changes. Therefore, in order for the behavior of iterations over the result of these methods to remain consistent with previous versions, an additional call to list() has to be performed in Python 3 to "take a snapshot" of the view object contents.

Reply:
remove list, keep it simple

Reply:
thanks for the explanation, that makes a lot of sense!